core/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java

```diff
@@ -462,6 +465,33 @@ public void simpleStats() throws Exception {
         assertThat(stats.getTotal().getIndexing(), nullValue());
         assertThat(stats.getTotal().getGet(), nullValue());
         assertThat(stats.getTotal().getSearch(), nullValue());
+
+        // index failed
+        try {
+            client().prepareIndex("test1", "type1", Integer.toString(1)).setSource("field", "value").setVersion(1)
+                    .setVersionType(VersionType.EXTERNAL).execute().actionGet();
+        } catch (VersionConflictEngineException e) {
+            //expected
+        }
+        try {
+            client().prepareIndex("test1", "type2", Integer.toString(1)).setSource("field", "value").setVersion(1)
+                    .setVersionType(VersionType.EXTERNAL).execute().actionGet();
+        } catch (VersionConflictEngineException e) {
+            //expected
+        }
+        try {
+            client().prepareIndex("test2", "type", Integer.toString(1)).setSource("field", "value").setVersion(1)
+                    .setVersionType(VersionType.EXTERNAL).execute().actionGet();
+        } catch (VersionConflictEngineException e) {
+            //expected
+        }
+        refresh();
```

Comment:
If its not required I'd drop it just to make it clear it isn't. If it is I'd put a comment about why because its unexpected that it would be.